code/mob/living/carbon.dm

```diff
@@ -27,7 +27,11 @@
 		if (!src.throwing && !src.lying && isturf(NewLoc))
 			var/turf/T = NewLoc
 			if (T.turf_flags & MOB_SLIP)
-				switch (T.wet)
+				var/wet_adjusted = T.wet
+				if (traitHolder && traitHolder.hasTrait("super_slips") && wet_adjusted == 1)
+					wet_adjusted = 2 //whee
```

Comment:
```suggestion
				if (traitHolder?.hasTrait("super_slips"))
					wet_adjusted = max(wet_adjusted, 2) //whee
```
This is cleaner imo and it shouldn't change anything unless wet is 0. But wet shouldn't be 0 on a MOB_SLIP turf, right?

Reply:
Turns out MOB_SLIP is all simulated floors and 0 is the default